test/extended/networking/acl_audit_log.go

```diff
@@ -0,0 +1,213 @@
+package networking
+
+import (
+	"context"
+	"fmt"
+	"strings"
+
+	exutil "github.com/openshift/origin/test/extended/util"
+	networkingv1 "k8s.io/api/networking/v1"
+
+	kapiv1 "k8s.io/api/core/v1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	e2e "k8s.io/kubernetes/test/e2e/framework"
+	frameworkpod "k8s.io/kubernetes/test/e2e/framework/pod"
+
+	. "github.com/onsi/ginkgo"
+	. "github.com/onsi/gomega"
+)
+
+var _ = Describe("[sig-network] acl-logging", func() {
+	oc := exutil.NewCLI("acl-logging")
+
+	// The OVNKubernetes subnet plugin should allow acl_logging for network policy.
+	// For third party plugins, the behavior is unspecified and we should not run either test.
+	InPluginContext([]string{},
+		func() {
+			It("should activate acl-logging", func() {
+				f := oc.KubeFramework()
+				makeNamespaceACLLoggingEnabled(oc, f.Namespace)
+				Expect(testACLLogging(f, oc)).To(Succeed())
```

Comment:
Don't Expect() a whole test, it will swallow all context / stack frames and make debugging difficult. Instead, structure your tests with nested Describe() and It().

For a reasonable test structure, check out the router or prometheus tests.

Reply:
Ack sounds good